source/copilot.txt

```diff
@@ -14,11 +14,11 @@
 
 |vsce-full| includes the |copilot| to assist in using `GitHub Copilot
 <https://github.com/features/copilot>`__ with your MongoDB deployments.
-Through GitHub Copilot's chat feature, users with |vsce| can interact
-with their MongoDB clusters and generate code with MongoDB
-domain-specific knowledge on top of GitHub Copilot's LLM. The |copilot|
-can also answer questions about your database collection schema and
-provide links to specific MongoDB documentation.  
+Through GitHub Copilot's chat feature, users with |vsce| can learn about
+MongoDB, interact with their MongoDB clusters, and generate code with
+MongoDB domain-specific knowledge on top of GitHub Copilot's LLM. The
+|copilot| can also answer questions about your database collection
```

Comment:
One general comment @ltran-mdb2 – could we switch the wording from MongoDB Github Copilot Participant to MongoDB Extension for Github Copilot? I just confirmed with the team at VS Code that participant isn't the proper branding (more of an implementation detail) and they want us to go with "MongoDB Extension" instead. Sorry for the change in wording!

Reply:
Done!